src/database/dummy-users.json

```diff
@@ -4,20 +4,42 @@
         "username": "Rihards-B",
         "password": "$2b$10$AeBAOtgGJxxRcUvrq/j7R.XSrGr9kXksjpxXzrOBDzvIQyx05YSj.",
         "first_name": "Rihards",
-        "last_name": "Blekus"
+        "last_name": "Blekus",
+        "roles": [
+            {
+                "_id": "66f13e501881fa27eb859e67",
```

Comment:
roels should only consist of the role name, there si no reason for the id to remain

Reply:
I still see that user objects contain role objects, the standard approach is for roles to just be string values in an array 

roles: [
'Admin',
'Manager',
'User'
]